types/cypress-cucumber-preprocessor/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for cypress-cucumber-preprocessor 1.12
+// Project: https://github.com/TheBrainFamily/cypress-cucumber-preprocessor
+// Definitions by: Alec Brunelle <https://github.com/aleccool213>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export {};
```

Comment:
This seems exceedingly incomplete, so this will be a regression for users in `noImplicitAny`.

At the very least, this module should provide `default` and `transform. `steps/index.d.ts` should define

- `given`
- `when`
- `then`
- `and`
- `but`
- `Given`
- `When`
- `Then`
- `And`
- `But`
- `defineParameterType`
- `defineStep`

Reply:
Good point :) I will add these.

Reply:
@DanielRosenwasser Checkout new commits.